test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/CompleteRulingSectionViewSpec.scala

```diff
@@ -18,13 +18,16 @@ package uk.gov.hmrc.tariffclassificationfrontend.views.partials
 
 import uk.gov.hmrc.tariffclassificationfrontend.forms._
 import uk.gov.hmrc.tariffclassificationfrontend.models.CaseStatus
+import uk.gov.hmrc.tariffclassificationfrontend.service.CommodityCodeService
 import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
 import uk.gov.hmrc.tariffclassificationfrontend.views.ViewSpec
 import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.complete_ruling_section
 import uk.gov.tariffclassificationfrontend.utils.Cases
 
 class CompleteRulingSectionViewSpec extends ViewSpec {
 
+  private val decisionForm = new DecisionForm(new CommodityCodeConstraints(new CommodityCodeService))
```

Comment:
Mocking?

Reply:
I've now mocked the CommodityCodeService